stetho/src/main/java/com/facebook/stetho/inspector/elements/android/FragmentDescriptor.java

```diff
@@ -64,6 +68,8 @@ protected int onGetChildCount(Object element) {
 
   @Override
   protected Object onGetChildAt(Object element, int index) {
+    verifyAccess();
```

Comment:
Also, what specifically are you worried about w.r.t. performance? If we look at the chain, `verifyAccess()` just calls `checkAccess()` (trivial to inline), which then calls `HandlerUtil.checkAccess()` (another trivial inline), which then calls two methods, `Looper.myLooper()` and `handler.getLooper()` (another trivial method). The only thing that seems worrisome here is that `Looper.myLooper()` makes use of thread-local storage via `ThreadLocal<T>.get()`. Is that what you're concerned about? I'm not yet familiar with the cost of TLS on Android.